packages/lib/markdownUtils.test.ts

```diff
@@ -0,0 +1,95 @@
+import markdownUtils from './markdownUtils';
+
+describe('Should detect list items', () => {
+	test('should detect `- lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('- lorem ipsum')).toBe(true);
+	});
+	test('should detect `+ lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('+ ')).toBe(true);
+	});
+	test('should detect `* lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('* lorem ipsum')).toBe(true);
+	});
+
+	// ordered list
+	test('should detect `1. lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('1. lorem ipsum')).toBe(true);
+	});
+	test('should detect `1) lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('1) lorem ipsum')).toBe(true);
+	});
+
+	test('should detect `+ [x] lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('1) lorem ipsum')).toBe(true);
+	});
+
+
+	test('should NOT detect `-lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('-lorem ipsum')).toBe(false);
+	});
+	test('should NOT detect `+lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('+')).toBe(false);
+	});
+	test('should NOT detect `*lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('*lorem ipsum')).toBe(false);
+	});
+
+	// ordered list
+	test('should NOT detect `1.lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('1.lorem ipsum')).toBe(false);
+	});
+	test('should NOT detect `1)lorem ipsum` as list item ', () => {
+		expect(markdownUtils.isListItem('1)lorem ipsum')).toBe(false);
+	});
+
+	test('should NOT detect `+ [x]lorem ipsum` as list item ', () => {
```

Comment:
It looks like you have a few copy and past typos here.

Reply:
Really sorry for  such a silly mistake,I fixed them in local repo.
should I remove this test on line 45 ?  since `+ [x]` is still a valid list item with `[x]` as the content of an unordered list.
![image](https://user-images.githubusercontent.com/63918341/113243210-34b40d00-92d0-11eb-92a2-ad288cb08a4a.png)

Reply:
Don't worry about it, these mistakes are really easy to make.